pkgs/development/python-modules/geoip2/default.nix

```
@@ -1,24 +1,29 @@
 { buildPythonPackage, lib, fetchPypi, isPy27
-, ipaddress
+, aiohttp
 , maxminddb
 , mock
+, mocket
 , requests
 , requests-mock
 }:
 
 buildPythonPackage rec {
-  version = "3.0.0";
+  version = "4.0.2";
   pname = "geoip2";
+  disabled = isPy27;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1q1ciqqd0qjkpgwsg9fws8wcqchkcq84gv2g4q3xgh2lpj3yjsaq";
+    sha256 = "06q9r5sdmncj4yaxrdf0mls05jb5n6pwhf8j8r74825cks4mvysa";
   };
 
-  propagatedBuildInputs = [ requests maxminddb ]
-    ++ lib.optionals isPy27 [ ipaddress ];
+  patchPhase = ''
+    substituteInPlace requirements.txt --replace "requests>=2.24.0" "requests>=2.23.0"
+  '';
```

```suggestion
  postPatch= ''
    substituteInPlace requirements.txt --replace "requests>=2.24.0" "requests"
  '';
```
Personal preference is not to lock versions here at all. The tests should fail if it doesn't work with that version.